src/main/rest-api/IdGenerator.js

```diff
@@ -0,0 +1,14 @@
+export class IdGenerator {
+    constructor (idLength = 6, charaters = 'abcdefghijklmnopqrstuvwxyz') {
+        this.idLength = idLength;
+        this.charaters = charaters;
+    }
+
+    generateId() {
+        let id = '';
+        for (let indexOfCharacter = 1; indexOfCharacter <= this.idLength; indexOfCharacter++) {
```

Comment:
I have a doubt. how you will guarantee that the id's doesn't repeat?

Reply:
I maked the set of possible characters larger so right now is harder to generate the same string, we can also increment the number of chars the id has

Reply:
That would be a good idea